Sources/JavaScriptKit/FundamentalObjects/JSFunction.swift

```diff
@@ -1,8 +1,8 @@
 import _CJavaScriptKit
 
-public class JSFunctionRef: JSObjectRef {
+public class JSFunctionRef: JSObject {
```

Comment:
Should this be `JSFunction`?
```suggestion
public class JSFunction: JSObject {
```

Reply:
Oops, I missed that. Thanks